### PR TITLE
Make 3.0.0-rc.1

### PR DESCRIPTION
**Pull request by kghbln**
This PR is made in reference to: #1365 

This PR addresses or contains:
- Proto version of RELEASE-NOTES
- Updated version of INSTALL.md
- Updated version of COMPATIBILITY.md

This PR includes:
- [ ] Tests (unit/integration)
- [ ] CI build passed

Fixes #

### CODE REVIEW

File: docs/COMPATIBILITY.md [hunk 1]
```diff
@@ -262,6 +268,14 @@ Note that MS SQL Server and Oracle are not supported as database backends.
...
 		<th><a href="http://rdf4j.org/">Sesame</a></th>
 		<th><a href="https://wiki.blazegraph.com/">Blazegraph</a></th>
+	</tr>
+		<tr>
+		<th>SMW 3.0.x</th>
+		<td>Full support<br />(1.x >=1.1) + 2.4.0</td>
+		<td>Full support<br />(6.x >=6.1) + 7.2<sup>[t.1]</sup></td>
+		<td>Beta support<br />(1.x >=1.1)<sup>[t.2]</sup></td>
+		<td>Full support<br />(2.8.x)</td>
+		<td>Full support<br />(1.5.2) + 2.1.0<sup>[t.3]</sup></td>
```
Comment (kghbln) [inline docs/COMPATIBILITY.md]: @mwjames Here I assumed that there are no compat changes with the rdf stores.

Comment (mwjames) [reply docs/COMPATIBILITY.md]: Remained the same.

**Comment by kghbln** [comment]
@mwjames If you could have a last peep at it. I am fully aware that the highlights section needs to be extended for the final release. I have the impression that we are missing out some major things here most notably ES.

**Comment by mwjames** [comment]
>If you could have a last peep at it. I am fully aware that the highlights section needs to be extended for the final release. 

There is still some time after the rc1 to fluff the content.

**Comment by mwjames** [comment]
I'd say go go go!

**Comment by kghbln** [comment]
> There is still some time after the rc1 to fluff the content.

Indeed. :)

**Comment by mwjames** [comment]
>  ... fully aware that the highlights section needs to be extended for the final release.

@kghbln To help you get started, I added some notes to [0]. You could also pick a topic do a two three day sprint, add the notes, and make a tweet hereby helps promote and create awareness while focusing on the main objective to get the release notes/highlights into a better shape.

[0] https://www.semantic-mediawiki.org/wiki/Semantic_MediaWiki_3.0.0/Highlights

**Comment by kghbln** [comment]
@mwjames Great, thanks a lot. Indeed still lots of emergency docu to do.

**Pull request merged**